client/src/components/Nav.jsx

```diff
@@ -0,0 +1,79 @@
+import React, {useState, useEffect} from 'react';
+import { Link } from 'react-router-dom';
+
+
+const Nav = (props) => {
+    const logout = async () => {
+    //   await fetch('http://localhost:8080/api/logout', {
+    //     method: 'POST',
+    //     headers: {'Content-Type': 'application/json'},
+    //     //credentials: 'include',
+    // });
+      
+    // clear the local storage if user logout
+      localStorage.clear();  
+      setUser('');
+      props.setFirstName('');
+    } 
+
+    const [user, setUser] =  useState(localStorage.getItem('profile'));
+    console.log("user " + user);
+    useEffect(() => {
+      const token = user?.token;
+      setUser(localStorage.getItem('profile'))
+    })
+
+    let menu;
+
+    if(user ==='' || user == null) {
+        menu = (
+          <ul className="navbar-nav me-auto mb-2 mb-md-0">
+              <li className="nav-item">
+                <Link to="/login" className="nav-link active">Login</Link>
+              </li>
+              <li className="nav-item">
+                <Link to="/register" className="nav-link active">Register</Link>
+              </li>
+          </ul>
+        )
+    } else {
+      menu = (
+        <ul className="navbar-nav me-auto mb-2 mb-md-0">
+              <li className="nav-item">
+                <Link to="/" className="nav-link active">Home</Link>
+              {/* Work in progress */}
```

Comment:
When logged in, there are two Home buttons on the navbar (one on the left like in logged out view, one on the right)

Reply:
You're right. @Lee-Taeho accidentally pushed his code to this branch and I didn't notice there are 2 Home buttons on the navbar. @Lee-Taeho which one do you think we should keep?

Reply:
I think we should replace the left home button with a CheckingN logo that @clantaca has provided in the discord. In that way, it would look closer to the original prototype